src/pages/Trade/components/OrderBookCard/MyTrades.tsx

```diff
@@ -1,50 +1,95 @@
-import React, { FC, useEffect } from 'react';
-import { connect, ConnectedProps } from 'react-redux';
-
-import {
-	fetchMyTradesRequest,
-	getMyTrades,
-	getIsLoadingMyTrades,
-	getIsLoadedMyTrades,
-} from 'ducks/trades/myTrades';
-
-import { RootState } from 'ducks/types';
-
-import useInterval from 'shared/hooks/useInterval';
+import React, { FC } from 'react';
+import { useQuery } from 'react-query';
+import snxData from 'synthetix-data';
+import { SYNTHS_MAP } from 'constants/currency';
 
 import TradeHistory from './TradeHistory';
 
 import { REFRESH_INTERVAL } from './constants';
+import { HistoricalTrades } from 'ducks/trades/types';
+import QUERY_KEYS from 'constants/queryKeys';
+import { CurrencyKey } from 'constants/currency';
 
-const mapStateToProps = (state: RootState) => ({
-	trades: getMyTrades(state),
-	isLoading: getIsLoadingMyTrades(state),
-	isLoaded: getIsLoadedMyTrades(state),
-});
-
-const mapDispatchToProps = {
-	fetchMyTradesRequest,
-};
+const MAX_TRADES = 100;
 
-const connector = connect(mapStateToProps, mapDispatchToProps);
+type MyTradesProps = {};
 
-type PropsFromRedux = ConnectedProps<typeof connector>;
+type SettledTrade = {
+	exchangeTimestamp: number;
+	dest: CurrencyKey;
+	src: CurrencyKey;
+	amount: number;
+	rebate: number;
+	reclaim: number;
+};
+type SettledTrades = SettledTrade[];
 
-type MyTradesProps = PropsFromRedux;
+const MyTrades: FC<MyTradesProps> = () => {
+	const currentWalletAddress = '0x2db6F5e838eD2BaD993E9FF2D3d7A5c1Cc35704C'; // TODO
```

Comment:
Should I continue to use redux to get the current wallet address or is there a `useQuery` key I can hook into?